website/docs/theme-classic.md

```diff
@@ -137,7 +137,9 @@ module.exports = {
           position: 'left', // or 'right'
           // To apply the active class styling on all
           // routes starting with this path.
+          // This usually isn't necessary
           activeBasePath: 'docs',
+          // activeBaseTest: 'docs/(next|v8)'
```

Comment:
Could we rename the option "activeBaseTest" with "activeBaseRegex"? It looks more understandable.